src/components/button-bar/button-bar.test.js

```diff
@@ -70,33 +70,6 @@ context("Test for Button-Bar component", () => {
         .and("eq", "1350px");
     });
 
-    describe("check children type for Button-Bar", () => {
```

Comment:
This now throws an actual error and is covered in the unit tests so I've just removed it here.